### ISSUE TITLE
make error 

### ISSUE THREAD

**Issue by KINGSABRI** [bararchy/sslscanner#1]
Hello there! 

I'm trying to build by as described in the readme. but I'm getting error

```
crystal build bin/scan.cr --release 
Error in bin/scan.cr:8: instantiating 'SSLScanner::Scan#run()'

scanner.run
        ^~~

in src/sslscanner.cr:14: instantiating 'gather_results()'

      sort_and_print(gather_results)
                     ^~~~~~~~~~~~~~

in src/sslscanner.cr:21: instantiating 'SSLScanner:Module#protocols()'

      SSLScanner.protocols.each do |symbol, protocol_bits|
                 ^~~~~~~~~

in src/sslscanner/ciphers.cr:2: undefined constant OpenSSL::SSL::Options::NO_SSLV3 (did you mean 'OpenSSL::SSL::Options::NO_SSL_V3')

  SSLV2   = [OpenSSL::SSL::Options::NO_SSLV3, OpenSSL::SSL::Options::NO_TLSV1, OpenSSL::SSL::Options::NO_TLSV1_1, OpenSSL::SSL::Options::NO_TLSV1_2]
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Makefile:63: recipe for target 'sslscanner' failed
make: *** [sslscanner] Error 1
```

OS: Ubuntu 17.10
Crystal: Crystal 0.24.2

**Comment by bararchy** [comment]
Seems the constant names had changed between releases 

```crystal
undefined constant OpenSSL::SSL::Options::NO_SSLV3 (did you mean 'OpenSSL::SSL::Options::NO_SSL_V3
```

I'll fix it in a few min unless you wana just PR the change

**Comment by bararchy** [comment]
@KINGSABRI Fixed via: 21011282b541022c717abe099afd18ec193ca9eb
